core/server/admin/app.js

```diff
@@ -38,7 +38,7 @@ module.exports = function setupAdminApp() {
     // @TODO ensure this gets a local 404 error handler
     adminApp.use('/assets', serveStatic(
         config.get('paths').clientAssets,
-        {maxAge: utils.ONE_YEAR_MS, fallthrough: false}
+        {maxAge: (config.get('env') === 'production') ? utils.ONE_YEAR_MS : 0, fallthrough: false}
```

Comment:
I would add this to production config (as we do for theme caching), because we would like to avoid having the env checks everywhere.